spec/expand_json_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "spec_helper"
 
 RSpec.describe Suspenders::Actions::ExpandJson do
-  let(:destination_root) { File.join(root_path, "tmp") }
+  let(:destination_root) { TestPaths.root_path.join("tmp") }
```

Comment:
This spec does not have explicit type metadata, so by default, we don't include any helper modules in it. That's why we're explicitly calling `TestPaths.root_path` instead of just `root_path`.